client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -21,110 +22,142 @@ const siteTypeOptions: Record< SiteType, string > = {
 };
 
 export default function OnboardingEdit() {
-	const [ verticalId, setVerticalId ] = useState< string >();
-
-	const { siteTitle, siteType } = useSelect( select => select( STORE_KEY ).getState() );
+	const [ lastSelectedQuestion, setLastSelectedQuestion ] = useState( null );
```

Comment:
Let's maybe rename to

```suggestion
	const [ focusedQuestion, setFocusedQuestion ] = useState( null );
```

? I think that might make reasoning a bit more intuitive.

Reply:
sounds good :+1: